packages/payments-plugin/e2e/mollie-payment.e2e-spec.ts

```diff
@@ -168,14 +183,25 @@ describe('Mollie payments', () => {
         expect(order.state).toBe('PaymentSettled');
     });
 
+    it('Should have Mollie metadata on payment', async () => {
+        const { order: { payments: [{ metadata }] } } = await adminClient.query(GET_ORDER_PAYMENTS, { id: order.id });
+        expect(metadata.mode).toBe(mockData.mollieResponse.mode);
+        expect(metadata.method).toBe(mockData.mollieResponse.method);
+        expect(metadata.profileId).toBe(mockData.mollieResponse.profileId);
+        expect(metadata.settlementAmount).toBe(mockData.mollieResponse.settlementAmount);
+        expect(metadata.customerId).toBe(mockData.mollieResponse.customerId);
+        expect(metadata.authorizedAt).toEqual(mockData.mollieResponse.authorizedAt.toISOString());
+        expect(metadata.paidAt).toEqual(mockData.mollieResponse.paidAt.toISOString());
+    });
+
     it('Should fail to refund', async () => {
         let mollieRequest;
         nock('https://api.mollie.com/')
             .post(/.*/, body => {
                 mollieRequest = body;
                 return true;
             })
-            .reply(200, { status: 'failed' });
+            .reply(200, { status: 'failed', resource: 'payment' });
```

Comment:
The new Mollie node client checks for the field `resource`. This is why the tests were failing after upgrading from Mollie 3.5 to 3.6